### DISCUSSION TITLE
What color theme is in the Readme?

### DISCUSSION THREAD

**Discussion by jeanlucthumm** [Bekaboo/dropbar.nvim#173] [Q&A] [answered]
README for this projects has beautiful light colorscheme:

![image](https://github.com/user-attachments/assets/412cc1b5-1b13-4356-b7cc-45b00d6a74f6)

Does anyone know which one this is? I thought it was Rose Pine but it doesn't quite match up

**Comment by Bekaboo** [comment, marked as answer]
It is a custom colorscheme of mine: https://github.com/Bekaboo/nvim/blob/master/colors/cockatoo.lua
